.vscode/settings.json

```diff
@@ -32,7 +32,6 @@
   "python.formatting.provider": "black",
   "python.linting.flake8Enabled": true,
   "python.linting.pylintEnabled": true,
-  "python.pythonPath": "${workspaceRoot}/.venv/bin/python",
```

Comment:
I guess you should leave this here.

Reply:
Sorry, VSCode keeps making this change on it's own and I committed it by accident.

Reply:
Changes made